holidayextras/ui-toolkit
# Convert last react classes

## What does this PR do?

This PR converts the last of the `React.createClass()` components to es2015 components and so should be compatible with React 15 and 16.

The build system uses babelify throughout rather than reactify.

## Testing
* Check the build passes
* Run the docs site with `npm run docs` and make sure the components are still rendering
* Build the dist folder with `npm run build` and copy the dist folder into tripapp's node_modules to check all the components also work there. Or ask and I can show you that working.

## Tech debt

I haven't bumped the version dependended on in package.json as this would be a major semantic version change and will also likely mean rewriting the whole test suite to use enzyme instead of the deprecated tools we're using now.

There's also one last use of `React.createClass()` in the documentation. It looks quite complicated a class so I've left it for now.
